### PR TITLE
LUN-394: Create user vocabulary screen

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by f1sh1918 — CHANGES_REQUESTED** [review]
Looks nice on ios. good job :). just some small adjustments

File: src/routes/UserVocabularyOverviewScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  margin: ${theme.spacings.xl};
+`
+
+interface PropsType {
+  navigation: StackNavigationProp<RoutesParams, 'UserVocabularyOverview'>
+}
+
+const UserVocabularyOverviewScreen = ({ navigation }: PropsType): JSX.Element => (
+  <Root>
+    <StyledHeading>{getLabels().ownVocabulary.myVocabulary}</StyledHeading>
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/routes/UserVocabularyOverviewScreen.tsx:31]: We do not really need it, but it makes automatic import possible, with just having the labels imported from json - at least for me - i always have to import it manually as intellij cannot do it automatically. So just makes my/our live easier.

**Comment by SteffiStuffel** [comment]
> Looks nice on ios. good job :). just some small adjustments

Thanks for prompt review. I would appreciate, if we only "request changes", if something bigger needs to be done / something is buggy or stuff like this. For only small adjustments that need to be done, approval or comment makes us work more efficently, i guess.

**Review by f1sh1918 — APPROVED** [review]

File: src/services/helpers.ts [hunk 2]
```diff
@@ -129,3 +129,5 @@ export const loadTrainingsSet = async (disciplineId: number): Promise<ServerResp
   const trainingSet = await getFromEndpoint<ServerResponseDiscipline>(trainingSetUrl)
   return trainingSet
 }
+
+export const getLabels = (): any => labels
```
Comment (steffenkleinle) [inline src/services/helpers.ts:133]: Doesn't this remove type annotations? Intellij currently tells me whether a label exists or does not.

**Review by steffenkleinle — APPROVED** [review]
Tested, works fine :)

File: src/services/helpers.ts (hunk 2 above)
Comment (SteffiStuffel) [reply src/services/helpers.ts:133]: You are right, added appropriate type.

**Pull request merged**